README.md

```
@@ -77,109 +106,226 @@
 ### 실시간 프리뷰 업데이트
+
 - 사용자의 카드 정보 입력에 따라 카드 프리뷰 화면이 실시간으로 동기화
 - 각 입력 필드의 변화가 즉시 카드 이미지에 반영
 
+### 동적 UI
+
+- 입력 필드는 사용자의 입력이 완료되면 다음 필드로 자동으로 이동한다.
+
 ## 폴더 구조
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)